cloudscan.py

```diff
@@ -224,26 +415,25 @@ def main():
             else:
                 ext_metadata = json.loads(options.ext_metadata)
             assert isinstance(ext_metadata, dict)
-        except:
-            print "External Metadata must be a dictionary!"
+        except Exception:
+            print("External Metadata must be a dictionary!")
             sys.exit(0)
     else:
         ext_metadata = dict()
 
     REQUEST_RETRIES = int(getConfig('request_retries'))
-    
+
     # Attempt to get the hostname
     try:
-        hostname = gethostname().split('.')[0] 
-    except:
+        hostname = gethostname().split('.')[0]
+    except Exception:
         hostname = "none"
 
-    
     # Attempt to set the return level, throw an error if it doesn't exist.
     try:
-        return_level = globals()["level_%s" % RETURN_LEVEL]
-    except KeyError as e:
-        print "Please specify a valid return level: minimal, metadata or full"
+        return_level = globals()["level_{0}".format(RETURN_LEVEL)]
```

Comment:
this is nothing short of janky.. is there a word for something worse than janky? that's what it is. but it was at least functional.. we need to add it back or refactor. up to you.